language/move-vm/runtime/src/loader.rs

```diff
@@ -2063,3 +2075,17 @@ impl Loader {
         self.type_to_type_layout_impl(ty, 1)
     }
 }
+
+// Public APIs for external uses.
+impl Loader {
+    pub fn get_type_layout(
```

Comment:
So excited to use this API in tooling!

Reply:
Yea I removed the other `Function` resolver api because that is already embeded in the `normalized::Function`. The code is a bit lengthy but at least I don't need to invoke Loader for that.